shipyard_blueprints/microsoft-excel/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "shipyard-excel"
-version = "0.1.0a1"
+version = "0.1.0"
 description = ""
 authors = ["wrp801 <wespoulsen@gmail.com>"]
 readme = "README.md"
```

Comment:
shipyard-microsoft-onedrive = "0.1.0a2"
shipyard-templates = "0.9.0a1"

should be bumped as well